stack.yaml

```diff
@@ -10,6 +10,7 @@ packages:
   - ./ouroboros-consensus
   - ./io-sim
   - ./io-sim-classes
+  - ./ntp-client
```

Comment:
Can you also add it to https://github.com/input-output-hk/ouroboros-network/blob/adf9378f7a4afee344ca6353a89ebfa152eb564a/release.nix#L21

I'm no Nix expert, but I see the components of this repo mentioned in some other nix files as well. Maybe they will be added when running `./nix/regenerate.sh`? We want to be sure that this component is also compiled by CI.

Reply:
It not ready yet to make it a required build but CI can already start building it.